src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -155,11 +158,15 @@ public void onGpgGameMessage(GpgGameMessageEvent event) {
 
       Path workDirectory = Paths.get(nativeDir).toAbsolutePath();
       String[] cmd = new String[]{
+          Paths.get(System.getProperty("java.home")).resolve("bin").resolve(org.bridj.Platform.isWindows() ? "java.exe" : "java").toAbsolutePath().toString(),
+          "-jar",
           getBinaryName(workDirectory),
           "--id", String.valueOf(currentPlayer.getId()),
           "--login", currentPlayer.getUsername(),
           "--rpc-port", String.valueOf(adapterPort),
-          "--gpgnet-port", String.valueOf(gpgPort)
+          "--gpgnet-port", String.valueOf(gpgPort),
+          "--log-directory", "iceAdapterLogs",
+          "--debug-window"
```

Comment:
`--debug-window` probably isn't desired for production, should be configurable, e.g. via environment variable

Reply:
for now that's desired, what do you mean via env variable? We will probable have to release another update once ICE is running stable. I will change the debug window though (e.g. tray icon) so that we don't scare the users.

The reason for keeping this is having fast access at it when stuff goes wrong over the next days and especially as I've sometimes seen the adapter not terminating so people can close it.

Reply:
This will stay for now (till the adapter is stable/proofed to be working), but the ice adapter will change so that the debug window will be a try icon or small info window by default.

Reply:
Here is a solution that is enabled by default but configurable: https://github.com/FAForever/downlords-faf-client/commit/ca51b1dc3a845cd6b371b6b04994a2b9287dc719